src/api/api.js

```diff
@@ -399,3 +399,11 @@ export const fetchEdgeSystem = () => {
     console.log(err);
   }
 };
+
+export const fetchEdgeEnforceGroups = () => {
+  try {
+    return instance.get(`${EDGE_API_BASE}/device-groups/enforce-edge-groups`);
+  } catch (err) {
+    console.log(err);
```

Comment:
What is the reason behind not using console.error here instead? I think if the request fails, it is a clear indication of a problem. Furthermore, Sentry won't catch this error because it watches the error messages in a browser.